src/backend/gporca/libgpos/src/_api.cpp

```diff
@@ -131,41 +131,41 @@ gpos_init(struct gpos_init_params *params)
 {
 	CWorker::abort_requested_by_system = params->abort_requested;
 
-	if (GPOS_OK != gpos::CMemoryPoolManager::Init())
+	GPOS_TRY
 	{
-		return;
-	}
+		gpos::CMemoryPoolManager::Init();
```

Comment:
The gpos::CMemoryPoolManager::Init() method is called here, but below the corresponding Shutdown() is called as CMemoryPoolManager::Shutdown(), i.e. without namespace specification. Now it is ok, but if someday a new CMemoryPoolManager class  appear in the default namespace, then the Shutdown() from another class would be called. I suggest to call Init() without namespace, because of "using namespace gpos;"

Reply:
done